content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst

```diff
@@ -2,44 +2,112 @@
+rule? Putaway is the process of taking products off the receiving shipments and directly putting
+them into the most appropriate location.
 
-If, for example, a warehouse contains volatile substances, it is important to make sure that certain products are not stored close to each other because of a potential chemical reaction. That’s where putaway rules intervene, to avoid storing products wrongly.
+If, for example, a warehouse contains volatile substances, it is important to make sure that
+certain products are not stored close to each other because of a potential chemical reaction.
+That's where putaway rules intervene, to avoid storing products wrongly.
 
 Configuration
 ==============
 
 In the *Inventory* app, go to :menuselection:`Configuration --> Settings` and activate the
 *Multi-Step Routes*. By doing so, the *Storage Locations* will be automatically activated.
 
-.. image:: putaway/putaw1.png
+.. image:: putaway/activate-multi-step-routes.png
    :align: center
+   :alt: Activate Multi-Step Routes in Inventory configuration settings.
 
 Setting up a Putaway Rule
 ==========================
 
-In some cases, like for a retail shop storing vegetables and fruits, we have to store products in different locations to maintain product quality. 
+In some cases, like for a retail shop storing vegetables and fruits, we have to store products in
+different locations depending on several factors like frequency, size, product category, need to be
+in specific environment etc...
 
-Let’s suppose there are one warehouse location *WH/Stock* and two sub-locations *WH/Stock/Vegetables* and *WH/Stock/Fruits*.
+Let's suppose there are one warehouse location *WH/Stock* with following sub-locations:
+
+- WH/Stock/Pallets
+  - WH/Stock/Pallets/PAL1
+  - WH/Stock/Pallets/PAL2
+  - WH/Stock/Pallets/PAL3
+- WH/Stock/Shelf 1
+- WH/Stock/Shelf 2
+  - WH/Stock/Shelf 2/Small Refrigerator
+- WH/Stock/Shelf 3
 
 To manage those locations, we will create putaway rules. To do so, open the *Inventory* app and go
 to :menuselection:`Configuration --> Putaway Rules`. Then, click on create and configure your first
 rule indicating the main location the product will enter before being redirected to the right
 location.
 
-.. note::
-   The putaway rules can be defined either per product or per product category.
+.. important::
+   The putaway rules can be defined either per product/product category and or Package Type
+   (Packages setting must be enabled for that). Putaway rules are read sequetially until a match is
+   found.
+
+Take the following example:
+
+- If I receive water (category All/drinks), whatever the package, it will be redirected to
+  WH/Stock/Shelf 2/Small Refrigerator
+- If I receive orange juice cans in a box, it will be redirected to WH/Stock/Shelf 2
+- If I receive water or apple juice in a box, it will be redirected to WH/Stock/Shelf 3
+- If I receive a Pallet of Lemonade cans, it will be redirected to WH/Stock/pallets/PAL1
```

Comment:
```suggestion
- If water (category All/drinks) is received, whatever the package, it will be redirected to
  WH/Stock/Shelf 2/Small Refrigerator.
- If orange juice cans, packaged in boxes, are received, they will be redirected to
  WH/Stock/Shelf 2.
- If water or apple juice bottles, packaged in boxes, are received, they will be redirected to
  WH/Stock/Shelf 3.
- If a pallet of lemonade cans are receieved, it will be redirected to WH/Stock/Pallets/PAL1.
```